packages/block-library/src/media-text/save.js

```diff
@@ -33,18 +34,17 @@ export default function save( { attributes } ) {
 		linkTarget,
 		rel,
 	} = attributes;
+	const mediaSizeSlug = attributes.mediaSizeSlug || DEFAULT_MEDIA_SIZE_SLUG;
 	const newRel = isEmpty( rel ) ? undefined : rel;
 
+	const imageClasses = classnames( {
+		[ `wp-image-${ mediaId }` ]: mediaId && mediaType === 'image',
+		[ `size-${ mediaSizeSlug }` ]: mediaId && mediaType === 'image',
+	} );
+	const imageClassName = imageClasses === '' ? null : imageClasses;
```

Comment:
#### Non-blocking:

Out of curiosity, does passing `null` instead of an empty string to className provide benefits?

Reply:
I presume is to avoid outputting the HTML `class` attributes altogether.
E.g.
- `<img className={ '' } />` -> `<img class />`
- `<img className={ null } />` -> `<img />`

It's probably not a big deal, but I guess we could just go with a simple `II` in the element itself 🤔 
```jsx
<img className={ imageClasses || null } />
```

Reply:
>I presume it is to avoid outputting the HTML class attributes altogether.
That makes sense 👍

> It's probably not a big deal
Agreed. This was definitely a smaller, non-blocking comment.

Reply:
Thanks! Changed it to `||`